extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -109,11 +150,43 @@ export class AmpSelector extends AMP.BaseElement {
         this.clearSelection_(element);
       }
     }
+    this.updateFocus_();
     // Update inputs.
     this.setInputs_();
   }
 
   /**
+   * Determine which option should receive focus first and set tabIndex
+   * on all options accordingly. When kb-select-mode is not 'none',
+   * the selector's focus should behave like focus on a set of radio buttons.
+   * In multi-select selectors, focus should just go to the first option.
+   * In single-select selectors, focus should go to the initially selected
+   * option, or to the first option if none are initially selected.
+   * @private
+   */
+  updateFocus_() {
+    if (this.kbSelectMode_ == KEYBOARD_SELECT_MODES.NONE) {
+      // Don't manage focus.
+      return;
+    }
+
+    this.options_.forEach(option => {
+      option.tabIndex = -1;
+    });
+
+    let initialFocusedElement;
+    if (this.isMultiple_) {
+      initialFocusedElement = this.options_[0];
```

Comment:
I had the same thought , i tried out checkboxes and this was the default behavior on them